pkgs/by-name/li/libmamba/package.nix

```diff
@@ -0,0 +1,44 @@
+{ fetchFromGitHub
+, lib
+, stdenv
+, cmake
+, fmt
+, spdlog
+, tl-expected
+, nlohmann_json
+, yaml-cpp
+, simdjson
+, reproc
+, libsolv
+, curl
+, libarchive
+, zstd
+, bzip2
+, python3
+, ...
+}:
+let
+  mamba-libsolv = libsolv.override { withConda = true; };
+in
+stdenv.mkDerivation rec {
+  nativeBuildInputs = [ cmake python3 ];
+  propagatedBuildInputs = [ fmt spdlog tl-expected nlohmann_json yaml-cpp simdjson reproc mamba-libsolv curl libarchive zstd bzip2 ];
```

Comment:
I'd rather have the list expanded. Overall, I think using `nixfmt-rfc-style` is probably not a bad idea regardless of one's preferences